### PR TITLE
feat(e2ee) add ability to disable the feature

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: modules/e2ee/E2EEncryption.js [hunk 1]
```diff
@@ -37,9 +37,12 @@ export class E2EEncryption {
             return false;
         }
 
-        return !(config.testing && config.testing.disableE2EE)
-            && (browser.supportsInsertableStreams()
-                || (config.enableEncodedTransformSupport && browser.supportsEncodedTransform()));
+        if (e2ee.disabled || config.testing?.disableE2EE) {
```
Comment (damencho) [inline modules/e2ee/E2EEncryption.js:40]: Adding this to config.js?

Comment (damencho) [reply modules/e2ee/E2EEncryption.js:40]: We need to merge the repos :)

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by damencho** [comment]
There is a known failure with PRs in checking jvb logs. @bgrozev will take a look, I think.

**Pull request merged**